projects/plugins/jetpack/modules/tiled-gallery/tiled-gallery/tiled-gallery-layout.php

```diff
@@ -53,6 +53,13 @@ abstract class Jetpack_Tiled_Gallery_Layout {
 	 */
 	public $columns;
 
+	/**
+	 * Attachment link
+	 *
+	 * @var string
```

Comment:
```suggestion
	 * @var bool
```

Reply:
Done in b503fca7d2b44b8c41de15263072a4145cc1b69c